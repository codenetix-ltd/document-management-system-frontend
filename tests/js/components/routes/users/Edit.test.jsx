import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import UserEdit from 'Components/routes/users/Edit';

describe('User edit', () => {
  let wrapper;
  let match;

  beforeEach(() => {
    match = {
      params: {
        userID: 1
      }
    };
    wrapper = shallow(<UserEdit match={match} />);
  });

  it('should render correctly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
