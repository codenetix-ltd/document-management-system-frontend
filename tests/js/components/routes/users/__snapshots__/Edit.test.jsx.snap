// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User edit should render correctly 1`] = `
<div>
  <ContentHeader
    breadcrumbs={
      Array [
        Object {
          "iconCls": "fa fa-users",
          "pageLink": "/users",
          "pageName": "Users",
        },
      ]
    }
    title="Edit "
  />
  <ContentWrapper
    boxClass="box-info"
    noBox={false}
  >
    <div
      className="box-header with-border"
    >
      <h3
        className="box-title"
      >
        Profile data
      </h3>
    </div>
    <UserForm
      getTemplateOptions={[Function]}
      onSubmit={[Function]}
      submitButtonText="Update"
      user={Object {}}
      validate={[Function]}
    />
  </ContentWrapper>
</div>
`;
